packages/synapse-react-client/src/components/EntityFinder/EntityFinder.tsx

```diff
@@ -188,114 +180,119 @@ export const EntityFinder: React.FunctionComponent<EntityFinderProps> = ({
     }
   }, [accessToken, searchTerms, handleError])
 
+  const handleTabChange = (
+    _event: React.SyntheticEvent,
+    newValue: EntityFinderTab,
+  ) => {
+    setSelectedEntitiesSnapshot(selectedEntities)
+    setCurrentTab(newValue)
+  }
+
   const mainPanelClass = searchActive
     ? 'MainPanelSearch'
     : treeOnly
     ? 'MainPanelSinglePane'
     : 'MainPanelDualPane'
 
-  let searchButtonText
-  const tableArray = [
-    'table',
-    'entityview',
-    'materializedview',
-    'submissionview',
-    'dataset',
-    'datasetcollection',
-  ]
-  if (selectableTypes.length === 1) {
-    searchButtonText = `Search for ${pluralize(
-      entityTypeToFriendlyName(selectableTypes[0]),
-    )}`
-  } else if (selectableTypes.every(element => tableArray.includes(element))) {
-    searchButtonText = 'Search for Tables'
-  } else {
-    searchButtonText = 'Search all of Synapse'
+  const entityDetailsListProps: EntityDetailsListProps = {
+    configuration: configFromTreeView,
+    versionSelection: versionSelection,
+    selected: selectedEntities,
+    isIdSelected: isIdSelected,
+    isSelectable: isSelectable,
+    visibleTypes: selectableAndVisibleTypesInList,
+    selectableTypes: selectableTypes,
+    selectColumnType: selectMultiple ? 'checkbox' : 'none',
+    toggleSelection: toggleSelection,
+    enableSelectAll: selectMultiple,
+    setCurrentContainer: setCurrentContainer,
   }
 
   return (
     <SynapseErrorBoundary>
       <div className="EntityFinder">
         <Box
           display="flex"
-          justifyContent="flex-end"
-          alignItems="stretch"
-          pb={1}
+          justifyContent="space-between"
+          mb={2.5}
+          flexWrap={'wrap'}
+          columnGap={16}
+          rowGap={2}
         >
```

Comment:
Updated this flex container to include the tabs & the search input. These are only styled using MUI, SCSS does not touch these